D1673/P1673.md

```diff
@@ -6069,97 +6069,103 @@ public:
     using rank_type = typename extents_type::rank_type;
     using layout_type = layout_blas_packed<Triangle, StorageOrder>;
 
```

Comment:
Should we consider section labels, as in [mdspan.layout.left]?

```suggestion

// [linalg.layouts.packed.cons], constructors
```

Reply:
We could.